src/main/java/api/data.java

```diff
@@ -0,0 +1,51 @@
+package api;
+
+import java.sql.DriverManager;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Scanner;
+
+//connect the SQL database file with java
+public class data {
```

Comment:
I'd recommend that you capitalize the name of your class to `Data`, or possibly a name that describes it reads from the database?

Reply:
Either `FetchFromDB` or `FetchFromDatabase` would match our current naming pattern.